package.json

```diff
@@ -47,8 +48,6 @@
     "mongodb": "1.4.39",
     "monk": "https://github.com/brave/monk.git",
     "monotonic-timestamp": "0.0.9",
-    "node-anonize2-relic": "https://github.com/brave/node-anonize2-relic.git",
```

Comment:
Good catch. I haven't wanted to bug you about it, but for some reason, that repo is broken for me in general (on Ubuntu and RHEL):
```
> node-anonize2-relic@0.1.3 install /usr/src/app/node_modules/node-anonize2-relic
> node preflight.js && node-gyp configure build

cmake failed
Error: spawn cmake ENOENT
```
I manually check it out and rebuild relic by hand to get node-anonize2-relic to run then the Dockerfile copies the prebuilt (which I forgot to check in here!). 

Have you ever seen this issue?